package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "katello",
+  "version": "1.0.0",
+  "scripts": {
+    "test": "jest webpack",
+    "test:watch": "jest webpack --watchAll",
+    "test:current": "jest webpack --watch",
+    "format":
+      "prettier --single-quote --trailing-comma es5 --write 'webpack/**/*.js'",
+    "build": "npm run format && npm run lint",
+    "lint": "eslint --fix webpack/index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/katello/katello.git"
+  },
+  "bugs": {
+    "url": "https://github.com/katello/katello/issues"
+  },
+  "devDependencies": {
+    "babel-jest": "^21.2.0",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "babel-preset-env": "^1.6.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "enzyme": "^3.1.0",
+    "enzyme-adapter-react-16": "^1.0.1",
+    "enzyme-to-json": "^3.1.2",
+    "eslint": "^4.8.0",
+    "eslint-config-airbnb": "^16.0.0",
+    "eslint-plugin-import": "^2.7.0",
+    "eslint-plugin-jest": "^21.2.0",
+    "eslint-plugin-jsx-a11y": "^6.0.2",
+    "eslint-plugin-react": "^7.4.0",
+    "jest": "^21.2.1",
+    "prettier": "^1.7.4",
+    "react-test-renderer": "^16.0.0"
+  },
+  "dependencies": {
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0",
+    "react-router-dom": "^4.2.2"
+  },
```

Comment:
Are `devDependencies` and `dependencies` something we can inherit from the foreman node package to stay on the same stack?

Reply:
@ehelms I was actually just thinking about that exact thing. It's a little bit of a weird scenario since foreman does it's own `npm install` and then runs `npm install` on the plugins. If we leave the dependencies in here it makes it a little more clear exactly what katello's dependencies are and it shouldn't conflict with foreman since we're allowing anything but a major version bump. Also `react-dom-router` is not included in Foreman's dependencies so we need that one for sure.

I'm up for thoughts anyone has about this.